packages/dht/src/connection/webrtc/BrowserWebrtcConnection.ts

```diff
@@ -30,14 +31,21 @@ export class NodeWebrtcConnection extends EventEmitter<Events> implements IWebrt
     private lastState: RTCPeerConnectionState = 'connecting'
     private readonly iceServers: IceServer[]
     private peerConnection?: RTCPeerConnection
-    private readonly bufferThresholdHigh = 2 ** 17
-    private readonly bufferThresholdLow = 2 ** 15
     private dataChannel?: RTCDataChannel
     private makingOffer = false
     private isOffering = false
     private closed = false
     private earlyTimeout: NodeJS.Timeout
     private readonly messageQueue: Uint8Array[] = []
+    private currentBufferedAmount = 0
+    private statistics: ConnectionStatistics = {
+        uploadRateBytesPerSecond: 0,
+        bufferedAmount: 0
+    }
+    private lastBytesSent = 0
+    private lastStatsCollectionTime = 0
+    private statsIntervalId?: NodeJS.Timeout
```

Comment:
could simply call this statsInterval or statsIntervalRef

Reply:
done